### PR TITLE
Added detailed instructions in README with gifs.

### PR DESCRIPTION
**Pull request by melissawm**
This is an experiment: not sure this is the right format.

### CODE REVIEW

**Comment by rgommers** [comment]
I like how that looks! Having the instructions in the README makes them very discoverable, and given that this is not the main repo I think focusing on contributions in the README is fine here. 

And I like the GIFs, better than a wall of text.

File: README.md [hunk 1]
```diff
@@ -51,15 +48,41 @@ used in the main NumPy documentation has two reasons:
...
+#### Create an issue
+
+Go to [https://github.com/numpy/numpy-tutorials/issues](https://github.com/numpy/numpy-tutorials/issues) and create a new issue with your proposal. Give as much detail as you can about what kind of content you would like to write (tutorial, how-to) and what you plan to cover. We will try to respond as quickly as possible with comments, if applicable.
+
+#### Upload your content
+
+Remember to clear all outputs on your notebook before uploading it. 
+
+**1. Fork this repository** (if you haven't before)
+<img src="images/01-fork.gif" width=80% height=80%>
```
Comment (bjnath) [inline README.md:62]: Make images collapsible.
```suggestion
<details>
<summary>
**1. Fork this repository** (if you haven't before)
</summary>
<img src="images/01-fork.gif" width=80% height=80%>
<details>
```

File: README.md [hunk 2]
```diff
@@ -51,15 +48,41 @@ used in the main NumPy documentation has two reasons:
...
+
+**2. In your own fork, create a new branch for your content**
+<img src="images/02-create_new_branch.gif" width=80% height=80%>
```
Comment (bjnath) [inline README.md:65]:
```suggestion
<details>
<summary>
**2. In your own fork, create a new branch for your content**
</summary>
<img src="images/02-create_new_branch.gif" width=80% height=80%>
</details>
```

File: README.md [hunk 3]
```diff
@@ -51,15 +48,41 @@ used in the main NumPy documentation has two reasons:
...
 
-1. Add your notebook to the `content/` directory
-2. Update the `environment.yml` file with the dependencies for your tutorial
+**3. Add your notebook to the `content/` directory**
+<img src="images/03-upload.gif" width=80% height=80%>
```
Comment (bjnath) [inline README.md:68]:
```suggestion
<details>
<summary>
**3. Add your notebook to the `content/` directory**
</summary>
<img src="images/03-upload.gif" width=80% height=80%>
</details>
```

File: README.md [hunk 4]
```diff
@@ -51,15 +48,41 @@ used in the main NumPy documentation has two reasons:
...
+
+**4. Update the `environment.yml` file with the dependencies for your tutorial**
 (only if you add new dependencies)
-3. Update this `README.md` to include your new entry
-4. Update the attribution section (below) to credit the original tutorial
-author.
+
+**5. Update this `README.md` to include your new entry**
+<img src="images/04-add_to_readme.gif" width=80% height=80%>
```
Comment (bjnath) [inline README.md:74]:
```suggestion
<details>
<summary>
**5. Update this `README.md` to include your new entry**
</summary>
<img src="images/04-add_to_readme.gif" width=80% height=80%>
</details>
```

File: README.md [hunk 5]
```diff
@@ -51,15 +48,41 @@ used in the main NumPy documentation has two reasons:
...
+
+**6. Update the attribution section (below) to credit the original tutorial
+author, if applicable.**
+
+**7. Create a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)**
+<img src="images/05-create_PR.gif" width=80% height=80%>
```
Comment (bjnath) [inline README.md:80]:
```suggestion
<details>
<summary>
**7. Create a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)**
</summary>
<img src="images/05-create_PR.gif" width=80% height=80%>
<details>
```

File: README.md (hunk 1 above)
Comment (melissawm) [reply README.md]: Thanks! Sorry for the late reply.

File: README.md [hunk 6]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
-1. Add your notebook to the `content/` directory
-2. Update the `environment.yml` file with the dependencies for your tutorial
+file) and you'd like to try add it out to the repository:
+
+#### Create an issue
+
+Go to [https://github.com/numpy/numpy-tutorials/issues](https://github.com/numpy/numpy-tutorials/issues) and create a new issue with your proposal. Give as much detail as you can about what kind of content you would like to write (tutorial, how-to) and what you plan to cover. We will try to respond as quickly as possible with comments, if applicable.
+
+#### Upload your content
+
```
Comment (bjnath) [inline README.md:58]:
```suggestion
<ul>
```

File: README.md [hunk 7]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
+Remember to clear all outputs on your notebook before uploading it. 
+
+<details>
+    <summary>
+        **1. Fork this repository** (if you haven't before)
```
Comment (bjnath) [inline README.md:63]:
```suggestion
        <b>Fork this repository</b> (if you haven't before).
```

File: README.md [hunk 8]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
+    </summary>
+    <img src="images/01-fork.gif" width=80% height=80%>
+</details>
+    
+<details>
+    <summary>
+        **2. In your own fork, create a new branch for your content**
```
Comment (bjnath) [inline README.md:70]:
```suggestion
        <b>In your own fork, create a new branch for your content.,</b>
```

File: README.md [hunk 9]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
+    </summary>
+    <img src="images/02-create_new_branch.gif" width=80% height=80%>
+</details>
+
+<details>
+    <summary>
+        **3. Add your notebook to the `content/` directory**
```
Comment (bjnath) [inline README.md:77]:
```suggestion
        <b>Add your notebook to the <code>content/</code> directory</b>.
```

File: README.md [hunk 10]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
+    </summary>
+    <img src="images/03-upload.gif" width=80% height=80%>
+</details>
+
+**4. Update the `environment.yml` file with the dependencies for your tutorial**
```
Comment (bjnath) [inline README.md:82]:
```suggestion
<b>Update the `environment.yml` file with the dependencies for your tutorial</b>
```

File: README.md [hunk 11]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
 (only if you add new dependencies)
```
Comment (bjnath) [inline README.md:83]:
```suggestion
(only if you add new dependencies).
```

File: README.md [hunk 12]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
-3. Update this `README.md` to include your new entry
-4. Update the attribution section (below) to credit the original tutorial
-author.
+
+<details>
+    <summary>
+        **5. Update this `README.md` to include your new entry**
```
Comment (bjnath) [inline README.md:87]:
```suggestion
        <b>Update this <code>README.md</code> to include your new entry.</b>
```

File: README.md [hunk 13]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
+    </summary>
+    <img src="images/04-add_to_readme.gif" width=80% height=80%>
+</details>
+
+**6. Update the attribution section (below) to credit the original tutorial
+author, if applicable.**
```
Comment (bjnath) [inline README.md:93]:
```suggestion
<b>Update the attribution section (below) to credit the original tutorial
author, if applicable.<b>
```

File: README.md [hunk 14]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
+
+<details>
+    <summary>
+        **7. Create a [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)**
```
Comment (bjnath) [inline README.md:97]:
```suggestion
        <b>Create a <a href="https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests"> pull request.</a></b>
```

File: README.md [hunk 15]
```diff
@@ -51,15 +48,61 @@ used in the main NumPy documentation has two reasons:
...
+    </summary>
+    <img src="images/05-create_PR.gif" width=80% height=80%>
+</details>
+
+:tada: **Wait for review!**
```
Comment (bjnath) [inline README.md:102]:
```suggestion
:tada: <b>Wait for review!</b>
</ul>

```

**Comment by bjnath** [comment]
My folding suggestion messed up the styling, which these revisions are meant to fix -- sorry.

The <ul> is to get the section indented (nothing gets bulleted).  

Added periods at the end of the sentences. 

Removed the numbering -- didn't mix well with the fold arrows.

**Comment by bjnath** [comment]
<details>
<summary>
Here is how it looks with the changes:
</summary>

</details>

**Comment by melissawm** [comment]
Love the way that looks, thanks!

**Comment by mattip** [comment]
This looks great. Should we merge as-is and iterate further as needed?

**Comment by melissawm** [comment]
I just added a last commit with more explicit instructions about the style guide. Should be good for merging unless there are any other comments.

**Review by rgommers — APPROVED** [review]
Very nice, let's give this a go!

**Pull request merged**